src/main/java/com/techiteasy/demo/mapping/CIModuleMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.techiteasy.demo.mapping;
+
+import com.techiteasy.demo.dto.CIModuleDto;
+import com.techiteasy.demo.dto.CIModuleInputDto;
+import com.techiteasy.demo.models.CIModule;
+
+public class CIModuleMapper {
+
+    // Entiteit naar DTO
+    public static CIModuleDto toCIModuleDto(CIModule ciModule) {
```

Comment:
Static mappers zijn ok, maar het is netter om `@`Component mappers te maken. Zonder static methods dus, maar je spreekt ze in de desbetreffende klasse aan via injection.